packages/humanize-distance/src/index.ts

```diff
@@ -1,27 +1,31 @@
-export function humanizeDistanceStringImperial(meters, abbreviate) {
```

Comment:
Is it necessary to change the file extension to `.ts`?

Reply:
I'm pretty sure it's necessary, if not useful at the very least for determining which files are typed and which aren't.